src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -13,13 +13,10 @@ def index
   end
 
   def update
-    notification = User.session.notifications.find(params[:id])
-    authorize notification, policy_class: NotificationPolicy
+    notifications = policy_scope(Notification).where(id: params[:notification_ids])
 
-    if notification.toggle(:delivered).save
-      flash.now[:success] = "Successfully marked the notification as #{notification.unread? ? 'unread' : 'read'}"
-    else
-      flash.now[:error] = "Couldn't mark the notification as #{notification.unread? ? 'read' : 'unread'}"
+    unless notifications.update_all('delivered = !delivered')
+      flash[:error] = "Couldn't mark the notifications as #{notifications.first.unread? ? 'read' : 'unread'}"
```

Comment:
We need to keep `.now` to not reintroduce the issue fixed in #10187.

```suggestion
      flash.now[:error] = "Couldn't mark the notifications as #{notifications.first.unread? ? 'read' : 'unread'}"
```

Reply:
Ahh, sorry this happend during the rebasing, was not by purpose

Reply:
No worries, it happens :slightly_smiling_face: